sdk/servicebus/service-bus/src/serializers/queueResourceSerializer.ts

```diff
@@ -31,7 +30,7 @@ import {
  * converts values to string and ensures the right order as expected by the service
  * @param queueOptions
  */
-export function buildQueueOptions(queueOptions: QueueOptions): InternalQueueOptions {
+export function buildQueueOptions(queueOptions: QueueDescription): InternalQueueOptions {
```

Comment:
queueOptions -> queue to mirror the user facing api where the input of type `QueueDescription` is named simply "queue". Same for topic, subscription and rule

Reply:
Done